openmrs-client/src/main/java/org/openmrs/mobile/models/Visit.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+
+package org.openmrs.mobile.models
+
+import com.google.gson.annotations.Expose
+
+class Visit : Resource() {
+
+    override var id: Long? = null
+
+    @Expose
+    var patient: Patient? = null
```

Comment:
```suggestion
    var patient: Patient
```
Do this for the rest of the class.